buildenv/jenkins/openjdk_tests

```diff
@@ -459,9 +459,9 @@ def runTest() {
         }
         jenkinsfile = load "${WORKSPACE}/aqa-tests/buildenv/jenkins/JenkinsfileBase"
         if (LABEL.contains('ci.agent.dynamic') && CLOUD_PROVIDER.equals('azure')) {
-            //Set dockerimage for azure agent. Fyre has stencil to setup the right environment
-            docker.image('adoptopenjdk/centos7_build_image').pull()
-            docker.image('adoptopenjdk/centos7_build_image').inside {
+            // Set dockerimage for azure agent. Fyre has stencil to setup the right environment
+            docker.image('ghcr.io/adoptium/test-containers:ubuntu2204').pull()
```

Comment:
Should we extract that URL or the ubuntu version?

Reply:
Referencing a set of containers from https://github.com/orgs/adoptium/packages/container/package/test-containers is good from my point of view, and we can build out several to give us a variety of coverage.